idp: add default icon urls to idps

mhilton opened the pull request:
Where appropriate set default icon urls for each idp. Also ensure that
the documentation includes notes about the icon configuration where it
may be used.

jujugui commented:
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/candid/81/

fabricematrat commented on store/mgostore/store.go:
```diff
@@ -244,3 +244,27 @@ func ensureIdentityIndexes(db *mgo.Database) error {
+// IdentityCounts implements store.Store.IdentityCounts.
+func (s *identityStore) IdentityCounts(ctx context.Context) (map[string]int, error) {
+	coll := s.b.c(ctx, identitiesCollection)
+	defer coll.Database.Session.Close()
```
Why do we need to Close here ?

mhilton commented:
:shipit:

jujugui commented:
Status: merge request accepted. Url: http://ci-gce.theblues.io:8080/job/candid-merge

mhilton committed:
idp: add default icon urls to idps

Where appropriate set default icon urls for each idp. Also ensure that
the documentation includes notes about the icon configuration where it
may be used.
